### DISCUSSION TITLE
9Go! [AU]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#897] [Q&A]
### Channel ID

9GoSydney.au

### Notes (optional)

Im making alot of requests because i cant watch my fav channels

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/au.m3u#L12-L13
